django_cloud_deploy/crash_handling/__init__.py

```diff
@@ -49,12 +50,22 @@ def handle_crash(err: Exception, command: str,
     # path of their Django project. We will put traceback in a public issue,
     # but user's might not want to put these kind of information to public,
     # so we plan to only handle crashes caused by our code.
+
+    log_fd, log_file_path = tempfile.mkstemp(
+        prefix='django-deploy-bug-report-')
     console.tell(
-        ('Your "{}" failed due to an internal error - please try again.'
+        ('Your "{}" failed due to an internal error.'
          '\n\n'
          'You can report this error by filing a bug on Github. If you agree,\n'
          'a browser window will open and an Github issue will be\n'
-         'pre-populated with the details of this crash.\n').format(command))
+         'pre-populated with the details of this crash.\n'
+         'See bug report at "{}" for more information.').format(
+             command, log_file_path))
+    issue_content = _create_issue_body(command)
```

Comment:
I'd write the file before telling the user about it.

Reply:
Done.